app/src/main/java/com/android/voyageur/model/trip/TripsViewModel.kt

```diff
@@ -181,6 +191,50 @@ open class TripsViewModel(
     }
   }
 
+  open fun getPhotosForSelectedTrip(): List<String> {
+    return selectedTrip.value?.photos ?: emptyList()
+  }
+
+  open fun addPhotoToTrip(photo: String) {
+    if (selectedTrip.value != null) {
```

Comment:
for better readability it might be better to use `if (selectedTrips.value == null) return` so there would be no need for nesting, but it's not a big deal.

Reply:
Good idea, it would also improve the code readability.